tor/helpers/flair.py

```diff
@@ -59,6 +59,32 @@ def _get_flair_css(transcription_count: int) -> str:
         return 'grafeas'
 
 
+def check_promotion(transcription_count):
```

Comment:
This is very similar to `_get_flair_css`, I wonder if we could combine them in some way?

Reply:
Sir I get what you are saying but I think if we combine them both then this function will definitely return something so I think we should'nt combine them both.